packages/analytics/analytics-chart/src/components/ZoomActions.vue

```diff
@@ -85,13 +85,22 @@ const handleAction = (option: ZoomActionItem) => {
     margin: 0 var(--kui-space-20, $kui-space-20);
   }
 
+  &:has(.zoom-actions-heading-divider + .zoom-action-item:hover) .zoom-actions-heading-divider {
+    display: none;
```

Comment:
nit: (Sorry in advance) This looks odd, the menu items jump ever so slightly now that this disappears. It may work better if the border-color changes to meet the background or something similar:

[Screencast_20250827_121651.webm](https://github.com/user-attachments/assets/d1ca3391-2506-4687-a632-41c6dca55478)